src/lando/conftest.py

```diff
@@ -572,11 +572,13 @@ def tag_factory(repo: Repo, seqno: int, commit: Commit) -> Tag:
 
 @pytest.fixture
 def make_push():
-    def push_factory(repo: Repo, commits: list[Commit]):
+    def push_factory(repo: Repo, commits: list[Commit] = [], tags: list[Tag] = []):
```

Comment:
You shouldn't use `[]` as a default in a keyword argument. Use `None` as the default and create the list inside the function if you need an empty list as the default instead.

Reply:
Yep. In this case, it's not at risk because I'm iterating over the elements of the list, rather than owning/modifying it, but fair enough.

Probably one of my biggest annoyance with Python 😅